buttercup/cogs/history.py

```diff
@@ -261,6 +256,121 @@ async def _history(
             file=discord_file,
         )
 
+    def get_user_rate(self, user: str) -> pd.DataFrame:
+        """Get a data frame representing the transcription rate of the user.
+
+        :returns: The rate data of the user.
+        """
+        # First, get the ID of the user
+        user_response = self.blossom_api.get_user(user)
+        if user_response.status != BlossomStatus.ok:
+            raise BlossomException(user_response)
+
+        user_id = user_response.data["id"]
+
+        # Get all rate data
+        user_data = self.get_all_rate_data(user_id, "day")
+
+        # Add an up-to-date entry
+        today = datetime.now(tz=tzutc()).replace(
+            hour=0, minute=0, second=0, microsecond=0
+        )
+        if today not in user_data.index:
+            user_data.loc[today] = [0]
+
+        return user_data
+
+    @cog_ext.cog_slash(
+        name="rate",
+        description="Display the rate graph.",
+        options=[
+            create_option(
+                name="users",
+                description="The users to display the rate graph for."
+                "Defaults to the user executing the command.",

```

Comment:
I'd say to specify in the description that we allow for a maximum of 5 users consecutively, both here and in the `history` command.

Reply:
Good point!